go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/smartcontractkit/chainlink/v2
 
-go 1.23
+go 1.23.3
```

Comment:
Is the patch version important? Seems odd to target 1.23.3 when 1.23.4 was just released.